source/2020-09-01-ember-3-21-released.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Ember 3.21 Released
+author: Kenneth Larsen, Isaac Lee
+tags: Releases, 2020, 3, 3.21, Version 3.x
+responsive: true
+---
+
+Today the Ember project is releasing version 3.21 of Ember.js, Ember Data, and Ember CLI.
+
+Version 3.20 of Ember is now promoted to LTS (Long Term Support). An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS version of Ember was 3.16.
+
+This release kicks off the 3.22 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.21
+
+Ember.js 3.21 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations.
+
+#### New Features (0)
+
+There are no new features introduced in Ember.js 3.21.
+
+#### Deprecations (1)
+
+1. `getWithDefault` (both the function and class method) has been deprecated per [RFC #554](https://github.com/emberjs/rfcs/blob/master/text/0554-deprecate-getwithdefault.md).
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+For more details on changes in Ember.js 3.21, please review the [Ember.js 3.21.0 release page](https://github.com/emberjs/ember.js/releases/tag/v3.21.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 3.21
+
+Ember Data 3.21 release saw a few improvements to documentation and continuous integration.
```

Comment:
I wasn't sure what https://github.com/emberjs/data/pull/7228 was about. If the change is important for developers to know, will you let me know how to explain the change?

Reply:
@igorT @runspired If you have time, can you help me with answering this question?